### PR TITLE
Consistent dataset names

### LINKED ISSUE #449: Inconsistent dataset naming publishing Taskcluster group
Follow-up of #406.
 
Label used from taskcluster is inconsistent among evaluations for runs and evaluations that does not have a run (thus displaying data in different graph):
* evaluate-student-flores-aug-typos_devtest-lt-en becomes "Flores-aug-typos_devtest-lt-en"
* evaluate-teacher-ensemble-flores-flores_aug-typos_devtest-lt-en becomes "Flores-flores_aug-typos_devtest-lt-en"

This can be fixed either by retrieving dataset name from the task, or handling the duplicate (e.g. `flores-flores…`).

### PR DESCRIPTION
**Pull request by vrigal**
Closes #449

### CODE REVIEW

**Comment by vrigal** [comment]
I published data from task group `FKykM4toScus1wnKQhDn_g`:
* [Run before](https://wandb.ai/teklia/494-before)
* [Run after](https://wandb.ai/teklia/494-after)

File: tests/test_tracking_utils.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import pytest
+
+from tracking.translations_parser.utils import parse_tag
+
+
+@pytest.mark.parametrize(
```
Comment (eu9ene) [inline tests/test_tracking_utils.py:6]: great job on adding tests!

**Review by eu9ene — APPROVED** [review]

**Comment by vrigal** [comment]
Uploads from other groups (which contains augmentation):
- group `ez3qMYf1T5O39n5TD7G79A` ([lt-en](https://wandb.ai/teklia/lt-en)).
- group `PLHimQugRzyDciwf8D5fEw` ([en-ru](https://wandb.ai/teklia/en-ru)).
- group `FKykM4toScus1wnKQhDn_g` ([en-ca](https://wandb.ai/teklia/en-ca)).

**Comment by eu9ene** [comment]
> Uploads from other groups (which contains augmentation):
> 
>     * group `ez3qMYf1T5O39n5TD7G79A` ([lt-en](https://wandb.ai/teklia/lt-en)).
> 
>     * group `PLHimQugRzyDciwf8D5fEw` ([en-ru](https://wandb.ai/teklia/en-ru)).
> 
>     * group `FKykM4toScus1wnKQhDn_g` ([en-ca](https://wandb.ai/teklia/en-ca)).

@vrigal I confirm that the dataset names look correct but why the experiment groups are missing? I see only the individual runs in the project:
Screenshot 2024-03-27 at 11 06 35 AM

**Comment by vrigal** [comment]
I was able to group runs on W&B, I think it is just a display option on their side:
Screenshot_2024-03-28_09-16-35

**Comment by eu9ene** [comment]
> I was able to group runs on W&B, I think it is just a display option on their side. I wonder why this is not the default option on the default workspace, I should rather share links with my own. Screenshot_2024-03-28_09-16-35

I see. Ok, I can see it properly now but we should figure out where this setting is set and make sure it's set by default. It's not trivial to find it.

**Pull request merged**